homeassistant/components/shelly/event.py

```diff
@@ -170,7 +207,7 @@ def __init__(
     ) -> None:
         """Initialize Shelly entity."""
         super().__init__(coordinator)
-        self.input_index = int(key.split(":")[-1])
+        self.id = int(key.split(":")[-1])
```

Comment:
Could we change this to `event_id`? We use `_id` in several places and it is always related to the component ID.
```suggestion
        self.event_id = int(key.split(":")[-1])
```